src/docs/spec.md

```diff
@@ -438,17 +448,19 @@ Illegal case 3:
 Example ([download](https://raw.githubusercontent.com/mapping-commons/sssom/master/examples/embedded/mp-hp-exact-0.0.1.sssom.tsv)):
 
 ```
-#creator_id: "https://orcid.org/0000-0002-7356-1779"
 #curie_map: 
 #  HP: "http://purl.obolibrary.org/obo/HP_"
 #  MP: "http://purl.obolibrary.org/obo/MP_"
-#  skos: "http://www.w3.org/2004/02/skos/core"
+#  skos: "http://www.w3.org/2004/02/skos/core#"
+#  orcid: "https://orcid.org/"
+#creator_id:
+#  - "orcid:0000-0002-7356-1779"
 #license: "https://creativecommons.org/publicdomain/zero/1.0/"
 #mapping_provider: "http://purl.obolibrary.org/obo/upheno.owl"
-subject_id	predicate_id	object_id	mapping_justification	subject_label	object_label
-HP:0009124	skos:exactMatch	MP:0000003	Lexical	Abnormal adipose tissue morphology	abnormal adipose tissue morphology
-HP:0008551	skos:exactMatch	MP:0000018	Lexical	Microtia	small ears
-HP:0000411	skos:exactMatch	MP:0000021	Lexical	Protruding ear	prominent ears
+subject_id  subject_label	predicate_id	object_id   object_label	mapping_justification
```

Comment:
I observe that the correction made here was to adhere to canonical column order.